NREL/boptest-dashboard
# Use typeorm for models

This PR kicks sequelize out in favor of typeorm. The current state defines all the models, and associates them. 

To test:
1. run docker-compose up with connectToDb(true) in index.ts
2. Once that successfully runs, change the param of connectToDb to false (this runs the quick hack to create some test data while I work on the endpoint and script to generate more solid test data). 
3. Hit http://localhost:8080/api/results to see a result with every other model connected to it (the basis of the entire boptest data model runs off the result object).
4. Send up an offering to ORM gods in thanks
